### PR TITLE
Fix barchart add port

### PR DESCRIPTION
**Pull request by ben-gready**
things were getting hairy, so I turned this into a PR

### CODE REVIEW

File: exec/runApplication.R [hunk 1]
```diff
@@ -7,9 +7,11 @@ parser <- ArgumentParser()
 # specify our desired options 
 parser$add_argument("--data_dir", type="character", default = "data/", 
                     help="Location of data store [default %(default)s]")
+parser$add_argument("--port", type="character", default = 3112,
```
Comment (reisner) [inline exec/runApplication.R]: would've preferred 3114, personally.

File: R/BarChartModules.R [hunk 2]
```diff
@@ -21,35 +21,39 @@ barChartView <-function(input, output, session, nlp_dt, input_text_dt){
...
-               xaxis = list(),
-               yaxis = list())
+                       xaxis = list(),
+                       yaxis = list())
     } else if(order_var == "mean_sentiment"){
-      plotly::plot_ly(data = word_freq, x = ~word,y = ~mean_sentiment, color = ~freq,name = "sentiment",type = "bar") %>%
+      word_freq = word_freq %>% dplyr::arrange(dplyr::desc(mean_sentiment)) %>% as.data.table
+      xform <- list(categoryorder = "array",
+                    categoryarray = word_freq$word)
+      plotly::plot_ly(data = word_freq %>%dplyr::arrange(dplyr::desc(mean_sentiment)) %>% as.data.table, x = ~word,y = ~mean_sentiment, color = ~freq,name = "sentiment",type = "bar") %>%
```
Comment (reisner) [inline R/BarChartModules.R]: new lines with each pipe please!

File: R/BarChartModules.R [hunk 3]
```diff
@@ -21,35 +21,39 @@ barChartView <-function(input, output, session, nlp_dt, input_text_dt){
...
+      dplyr::ungroup() %>%
+      dplyr::filter(freq >= input$min_freq)# %>%
+    
     xform <- list(categoryorder = "array",
                   categoryarray = word_freq$word)
     if(order_var == "freq"){
+      word_freq = word_freq %>% dplyr::arrange(dplyr::desc(freq)) %>% as.data.table
+      xform <- list(categoryorder = "array",
+                    categoryarray = word_freq$word)
       plotly::plot_ly(data = word_freq, x = ~word,y = ~freq, color = ~mean_sentiment,name = "word freq",type = "bar") %>%
```
Comment (reisner) [inline R/BarChartModules.R]: Do we need the word "sentiment" or "word freq" when hovering?

sent

**Comment by ben-gready** [comment]
thanks for the comments - please see fixes + a couple of additional things 1) x labels on bar chart now don't disappear at bottom 2) I've removed dependency on `relevance` variable in timeline plot

**Comment by reisner** [comment]
@ben-gready I get weird info when hovering over the bars now:

screenshot at feb 20 08-50-46

screenshot at feb 20 08-51-43

**Comment by reisner** [comment]
Looks good to me

**Pull request merged**